barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/grand_product_library.test.cpp

```diff
@@ -47,33 +47,15 @@ template <class FF> class GrandProductTests : public testing::Test {
      */
```

Comment:
Lots of cleanup in this test. For some reason it used to instantiate only the polynomials needed for the given operation under test. There's no advantage to this so now I just construct an entire ProverPolynomials. Probably this is just a relic from the days before this was easy.